### PR TITLE
Reporting feature - first few tests implemented

### PR DESCRIPTION
**Pull request by lhellebr**
This depends on https://github.com/SatelliteQE/airgun/pull/298

### CODE REVIEW

File: tests/foreman/ui_airgun/test_reporttemplates.py [hunk 1]
```diff
@@ -146,6 +154,39 @@ def test_positive_unlock_report(session):
...
+    name = gen_string('alpha')
+    clone_name = gen_string('alpha')
+    content = gen_string('alpha')
+    with session:
+        session.reporttemplate.create({
+            'template.name': name,
+            'template.template_editor.editor': content,
+        })
+        assert session.reporttemplate.search(name)[0]['Name'] == name
+        session.reporttemplate.clone(
```
Comment (omkarkhatavkar) [inline tests/foreman/ui_airgun/test_reporttemplates.py]: Can you please add assertion on content as well? (equally important )

File: tests/foreman/ui_airgun/test_reporttemplates.py [hunk 2]
```diff
@@ -146,6 +154,39 @@ def test_positive_unlock_report(session):
...
+            name,
+            {
+                'template.name': clone_name,
+            }
+        )
+        assert session.reporttemplate.search(
+            clone_name)[0]['Name'] == clone_name
+
```
Comment (omkarkhatavkar) [inline tests/foreman/ui_airgun/test_reporttemplates.py]: same here Can you please add assertion on content as well? (equally important )  

also i was thinking to check assert session.reporttemplate.search(name)[0]['Name'] == name after clone works fine, what you think ?

File: tests/foreman/ui_airgun/test_reporttemplates.py (hunk 1 above)
Comment (lhellebr) [reply tests/foreman/ui_airgun/test_reporttemplates.py]: This is a cloning test so I won't assert before cloning at all

**Comment by lhellebr** [comment]
```
========================================= test session starts ==========================================
platform linux -- Python 3.6.8, pytest-3.6.1, py-1.7.0, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/lhellebr/git/robottelo, inifile:
plugins: services-1.3.0, mock-1.10.0
collecting 1 item                                                                                      2019-03-28 12:32:03 - conftest - DEBUG - Collected 1 test cases

collected 1 item                                                                                       

tests/foreman/ui_airgun/test_reporttemplates.py .                                                [100%]

====================================== 1 passed in 120.11 seconds ======================================
```

**Comment by omkarkhatavkar** [comment]
> ```
> ========================================= test session starts ==========================================
> platform linux -- Python 3.6.8, pytest-3.6.1, py-1.7.0, pluggy-0.6.0
> shared_function enabled - OFF - scope:  - storage: file
> rootdir: /home/lhellebr/git/robottelo, inifile:
> plugins: services-1.3.0, mock-1.10.0
> collecting 1 item                                                                                      2019-03-28 12:32:03 - conftest - DEBUG - Collected 1 test cases
> 
> collected 1 item                                                                                       
> 
> tests/foreman/ui_airgun/test_reporttemplates.py .                                                [100%]
> 
> ====================================== 1 passed in 120.11 seconds ======================================
> ```

@lhellebr  looks good!

**Comment by omkarkhatavkar** [comment]
Merging this as got both ACK and AIrgun code also got merged.

**Pull request merged**